src/bundle/Resources/views/themes/admin/ui/tab/system_info.html.twig

```diff
@@ -1,26 +1,13 @@
-<div class="row ez-header pt-3">
-    <div class="container ez-container">
-        <ul class="nav nav-tabs ez-tabs nav-tabs-{{ group }}" role="tablist" id="ez-tab-list-{{ group }}">
-            {% for tab in tabs %}
-                {% set id = group ~ '-' ~ tab.identifier %}
-                {% set active = loop.first %}
-                <li class="nav-item">
-                    <a class="nav-link{% if active %} active{% endif %}" id="ez-tab-label-{{ group }}"
-                       data-toggle="tab" href="#ez-tab-{{ id }}" role="tab" aria-controls="ez-tab-{{ id }}"
-                       aria-expanded="{{ active }}">{{ tab.name }}</a>
-                </li>
-            {% endfor %}
-        </ul>
-    </div>
-</div>
-<div class="tab-content container ez-container" id="ez-tab-list-content-{{ group }}">
-    {% for tab in tabs %}
-        {% set id = group ~ '-' ~ tab.identifier %}
-        {% set active = loop.first %}
-
-        <div class="tab-pane{% if active %} active{% endif %}" id="ez-tab-{{ id }}" role="tabpanel"
-             aria-labelledby="ez-tab-label-{{ group }}">
-            {{ tab.view|raw }}
-        </div>
-    {% endfor %}
-</div>
+{% set tabs_to_show = [] %}
+{% for tab in tabs %}
```

Comment:
I think it would be cleaner to use map here
```
set tabs_to_show = tabs|map((tab, index) => {
        id: group ~ '-' ~ tab.identifier,
        label: tab.name,
        content: tab.view,
        active: index == 0,
    })
```